protoc-gen-go-grpc.hcl

```diff
@@ -0,0 +1,12 @@
+description = "protoc-gen-go-grpc is the Go implementation of gRPC: A high performance, open source, general RPC framework that puts mobile and HTTP/2 first."
+binaries = ["protoc-gen-go-grpc"]
+test = "protoc-gen-go-grpc --version"
+source = "https://github.com/grpc/grpc-go/releases/download/cmd/protoc-gen-go-grpc/v${version}/protoc-gen-go-grpc.v${version}.${os}.${arch}.tar.gz"
+strip = 1
+
+version "1.1.0" {
+  auto-version {
+    github-release = "grpc/grpc-go"
+    version-pattern = "cmd/protoc-gen-go-grpc/v(.*)"
```

Comment:
Did you try this out?

Reply:
ehm, how can I try it please?

Reply:
fwiw, this the release action that uploads artefacts:
https://github.com/grpc/grpc-go/blob/cmd/protoc-gen-go-grpc/v1.1.0/.github/workflows/release.yml#L11

Reply:
`hermit -d manifest auto-version *.hcl` ... ok, looking.

Reply:
:boom:
```
debug: Auto-versioning /Users/juliao/Development/hermit-packages/protoc-gen-go-grpc.hcl
fatal: grpc/grpc-go: latest release must match the pattern cmd/protoc-gen-go-grpc/v(.*) but is v1.40.0
github.com/cashapp/hermit/manifest.autoVersion
	/home/runner/work/hermit/hermit/manifest/autoversion.go:84
github.com/cashapp/hermit/manifest.AutoVersion
	/home/runner/work/hermit/hermit/manifest/autoversion.go:23
github.com/cashapp/hermit/app.(*autoVersionCmd).Run
	/home/runner/work/hermit/hermit/app/commands.go:1031
	...
```

Reply:
Maybe I'll just take the auto-versioning out until we can fix this is hermit.

Reply:
Done. Unless you have a better idea?

Reply:
Aside: How's that going to pan out for the next jq release candidate?

Reply:
I thought version-pattern was to filter out tags/releases, but I guess I must have misunderstood.
> Regex with one capture group to extract the version number from the origin.

I see.

Reply:
The default pattern will match the tag...

Reply:
ie. if you leave the pattern override out everything should work fine 👌

Reply:
so protoc-gen-go-grpc cuts releases quite regularlybut ONLY for the module / libs not the actual command (protoc plugin). See https://github.com/grpc/grpc-go/releases - no artefacts attached.
=> I don't think this will work with autoversioning.